src/parser/Parser/Parser.hs

```diff
@@ -531,7 +539,7 @@ optionalTypeParameters = option [] (brackets $ commaSep1 modedTypeVar)
 
 typedef :: EncParser Typedef
 typedef = do
-  typedefmeta <- meta <$> getPosition
+  typedefmeta <- meta . newPos <$> getPosition
```

Comment:
It seems that this pattern (`meta . newPos`) creates a new `Position` to wrap it inside a `Meta`. Does it make sense to create a function that returns the `Meta` from a `Position`? I think this is less cognitive load, as I don't care too much about how I got the `meta` from a `Position`